exporters/zipkin/src/test/java/io/opentelemetry/exporter/zipkin/ZipkinTestSpan.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.zipkin;
+
+import static org.mockito.Mockito.mock;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.TraceFlags;
+import io.opentelemetry.api.trace.TraceState;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.testing.trace.TestSpanData;
+import io.opentelemetry.sdk.trace.data.EventData;
+import io.opentelemetry.sdk.trace.data.StatusData;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+import java.net.InetAddress;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import zipkin2.Endpoint;
+import zipkin2.Span;
+
+class ZipkinTestSpan {
+
+  static final String TRACE_ID = "d239036e7d5cec116b562147388b35bf";
+  static final String SPAN_ID = "9cc1e3049173be09";
+  static final String PARENT_SPAN_ID = "8b03ab423da481c5";
+
+  static final InetAddress localAddressForTesting = mock(InetAddress.class);
```

Comment:
It's strange to have a utility class with a shared static mock. Better to have the static methods accept an `InetAddress`.

Reply:
I'm not clear on why you think it's strange.  It's intended to just be a constant that's referenced between the mocks/setups and the assertions. I can create a new instance in each test I suppose, but then it will have to be passed around a _lot_.

Reply:
Well I went ahead and removed the static/shared ip address anyway. 🤷🏻